cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -246,9 +246,9 @@ phases:
             - |
               set -v
               mkdir -p /etc/chef && sudo chown -R root:root /etc/chef
-              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz {{ build.CookbookUrl.outputs.stdout }}
-              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz.date {{ build.CookbookUrl.outputs.stdout }}.date
-              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz.md5 {{ build.CookbookUrl.outputs.stdout }}.md5
+              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz "{{ build.CookbookUrl.outputs.stdout }}"
+              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz.date "{{ build.CookbookUrl.outputs.stdout }}.date"
```

Comment:
I'm not sure these to link (for .date and .md5) are working this way if the `build.CookbookUrl.outputs.stdout` is a presigned URL

Reply:
You're right. Actually it looks like md5 and date file are not used. What about keeping it simple and avoid to download them?

Reply:
I ended up removing md5 and date as they were unused